app/hydrator/templates/create/toppanel.html

```diff
@@ -17,7 +17,7 @@
 <div class="side-panel top text-center">
   <!-- change to ng-if="true" to get this to render, this is WIP for the overall toppanel react migration -->
   <top-panel-name-and-description
-    ng-if="false"
+    ng-if="true"
```

Comment:
Better flip them back before merge?

Reply:
nah this is fine since we're merging it into that branch

Reply:
Oh, correct